appyters/ChIP_seq/templates/base.j2

```diff
@@ -1,6 +1,6 @@
 {% extends "biojupies/base.j2" %}
 {% block head %}
 <title>ChIP-seq analysis pipeline</title>  
-<link rel="shortcut icon" href="https://image.flaticon.com/icons/svg/65/65987.svg"/>
-<link rel="icon" href="https://image.flaticon.com/icons/svg/65/65987.svg">  
+<link rel="shortcut icon" href="../static/analysis.png"/>
+<link rel="icon" href="../static/analysis.png">  
```

Comment:
@mjjeon I'll change these for you but please use `{{ url_for('static', 'analysis.png') }}` instead in the future; this mechanism will ensure that these relative paths function regardless of the deployment strategy (for instance, if they are **not submounted onto a fixed prefix**